java/java-ranger-regression/WBS/WBS_prop1.yml

```diff
@@ -0,0 +1,8 @@
+format_version: "1.0"
+input_files:
+  - ../../common/
+  - impl/
```

Comment:
It feels like the `java/java-ranger-regression/WBS/prop1/Main.java`
should be included here as well, if I am not totally mistaken.

Reply:
Done. I was misunderstanding this `input_files` directive but I've now added `prop1/` as an `input_files` entry.